console/src/components/Main/Main.js

```diff
@@ -35,10 +35,40 @@ import {
   setProClickState,
   getLoveConsentState,
   setLoveConsentState,
+  getUserType,
 } from './utils';
 import { getSchemaBaseRoute } from '../Common/utils/routesUtils';
 import LoveSection from './LoveSection';
 import { Help, ProPopup } from './components/';
+import { HASURA_COLLABORATOR_TOKEN } from '../../constants';
+import { UPDATE_CONSOLE_NOTIFICATIONS } from '../../telemetry/Actions';
+
+const updateRequestHeaders = props => {
+  const { requestHeaders, dispatch } = props;
+
+  const collabTokenPresent = Object.keys(requestHeaders).find(
+    hdr => hdr.toLowerCase() === HASURA_COLLABORATOR_TOKEN
+  );
+
+  if (collabTokenPresent) {
+    const userID = getUserType(collabTokenPresent.value);
```

Comment:
This line is causing a crash probably due to `collabTokenPresent` being a string object and `.value` doesn't exist